draft-ietf-pquip-pqc-engineers.md

```diff
@@ -334,31 +342,30 @@ KEM relies on the following primitives [PQCAPI]:
 * def kemEncaps(pk) -> (ct, ss)
 * def kemDecaps(ct, sk) -> ss
 
-where pk is public key, sk is secret key, ct is the ciphertext representing an encapsulated key, and ss is shared secret.  The following figure illustrates a sample flow of KEM:
+where pk is public key, sk is secret key, ct is the ciphertext representing an encapsulated key, and ss is shared secret.  The following figure illustrates a sample flow of KEM based key exchange:
 
 ~~~~~ aasvg
-
```

Comment:
The new diagram is not consistent with the KEM APIs mentioned above the diagram: kemKeyGen, kemEncaps and kemDecaps.

Reply:
It is, the diagram has kemKeyGen, kemEncaps and kemDecaps